Makefile

```diff
@@ -219,7 +228,8 @@ endif
 .PHONY: e2e-linux-amd64 e2e-linux-arm64 e2e-darwin-amd64 e2e-windows-amd64.exe
 e2e-linux-amd64: out/e2e-linux-amd64 ## Execute end-to-end testing for Linux 64bit
 e2e-linux-arm64: out/e2e-linux-arm64 ## Execute end-to-end testing for Linux ARM 64bit
-e2e-darwin-amd64: out/e2e-darwin-amd64 ## Execute end-to-end testing for Darwin 64bit
+e2e-darwin-amd64: out/e2e-darwin-amd64 ## Execute end-to-end testing for Darwin x86 64bit
```

Comment:
lets fix the comment for all e2e,  I think the right comment should be

"build end2end binary for ..." (it doesnt execute the test)

Reply:
done